bigtable/Dockerfile

```diff
@@ -0,0 +1,22 @@
+## Build
+FROM golang:1.11-buster AS build
+
+WORKDIR /app
```

Comment:
We probably don't need this? I think you can just drop the file into root.  `/app` is a thing we use in our internal cogs

Reply:
not really, I can change to put it in the root. this container will not be the final one since its only used for building

Reply:
Oh true, doesn't really matter then